Tools/autotest/arduplane.py

```diff
@@ -529,6 +530,69 @@ def fly_flaps(self):
         if ex:
             raise ex
 
+    def test_throttle_failsafe(self):
+        self.context_push()
+        ex = None
+        try:
+            self.mavproxy.send('switch 6\n')
+            self.wait_mode('MANUAL')
+            m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+            receiver_bit = mavutil.mavlink.MAV_SYS_STATUS_SENSOR_RC_RECEIVER
+            self.progress("Testing receiver enabled")
+            if (not (m.onboard_control_sensors_enabled & receiver_bit)):
+                raise PreconditionFailedException()
+            self.progress("Testing receiver present")
+            if (not (m.onboard_control_sensors_present & receiver_bit)):
+                raise PreconditionFailedException()
+            self.progress("Testing receiver health")
+            if (not (m.onboard_control_sensors_health & receiver_bit)):
+                raise PreconditionFailedException()
+            self.progress("Failing receiver")
+            self.set_parameter("SIM_RC_THRFAIL", 1)
+            self.wait_mode('CIRCLE') # short failsafe
+            self.wait_mode('RTL') # long failsafe
+            m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+            print("%s" % str(m))
+            m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+            print("%s" % str(m))
+            m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+            print("%s" % str(m))
+            m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+            print("%s" % str(m))
+            m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+            print("%s" % str(m))
+            m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+            print("%s" % str(m))
```

Comment:
Do we really need to print the entire SYS_STATUS message 5 times?

Reply:
Apparently it was useful - probably to see how the value changed over time.